### PR TITLE
refactor(android): Make app links more robust in the emulator

### PR DESCRIPTION
**Pull request by jamilbk**
Getting some weird behavior with AppLinks. They don't seem to work upon first use and require a few tries to function correctly.

Edit: Found the issue: Android Studio doesn't like when the Manifest contains variables for AppLinks. I added a note in the Manifest.

@conectado To test Applinks are working correctly, you can use the App Link Assistant:

Screenshot 2023-09-28 at 11 15 11 PM

Then from there you can click "Test App Links":

Screenshot 2023-09-28 at 11 15 30 PM

### CODE REVIEW

File: elixir/config/config.exs [hunk 1]
```diff
@@ -109,9 +109,9 @@ config :web,
   private_clients: [%{__struct__: Postgrex.INET, address: {172, 28, 0, 0}, netmask: 16}]
 
 config :web, Web.Auth,
-  platform_redirect_urls: %{
-    "apple" => "firezone://handle_client_auth_callback",
-    "android" => "https://app.firez.one/handle_client_auth_callback"
+  platform_redirects: %{
+    "apple" => [method: :external, dest: "firezone://handle_client_auth_callback"],
+    "android" => [method: :to, dest: "/handle_client_auth_callback"]
```
Comment (jamilbk) [inline elixir/config/config.exs:114]: This format allows us to debug locally.

File: docker-compose.yml [hunk 2]
```diff
@@ -3,7 +3,7 @@ version: '3.8'
 services:
   # Dependencies
   postgres:
-    image: postgres:15
+    image: postgres:16
```
Comment (jamilbk) [inline docker-compose.yml:6]: 🤷‍♂️

**Review by pratikvelani — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (jamilbk) [reply docker-compose.yml:6]: I figured tracking the latest stable will allow us to get ahead of any incompatibilities quickly. WDYT @AndrewDryga? Better to track the version on GCP?

https://www.postgresql.org/about/news/postgresql-16-released-2715/

File: elixir/apps/web/test/web/controllers/auth_controller_test.exs [hunk 3]
```diff
@@ -258,8 +258,6 @@ defmodule Web.AuthControllerTest do
 
       assert redirected_to = redirected_to(conn)
       assert redirected_to_uri = URI.parse(redirected_to)
-      assert redirected_to_uri.scheme == "https"
-      assert redirected_to_uri.host == "app.firez.one"
```
Comment (AndrewDryga) [inline elixir/apps/web/test/web/controllers/auth_controller_test.exs:262]: We are missing tests for both of behaviours to make sure we do a full redirect on apple and path redirect on android, other than that LGTM.

Comment (jamilbk) [reply elixir/apps/web/test/web/controllers/auth_controller_test.exs:262]: Good catch. Will add these.

Comment (jamilbk) [reply elixir/apps/web/test/web/controllers/auth_controller_test.exs:262]: Added!

**Pull request merged**